src/v3/test/integration/__snapshots__/error-400-unauthorized-client.test.tsx.snap

```diff
@@ -41,12 +42,12 @@ exports[`error-400-unauthorized-client should render form 1`] = `
                       <svg
                         aria-hidden="true"
                         class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-7"
-                        data-testid="InfoOutlinedIcon"
+                        data-testid="ErrorOutlineIcon"
```

Comment:
remove data-testid?

Reply:
Same as here: https://github.com/okta/okta-signin-widget/pull/2617#discussion_r930543255

I think it was caused by me updating the class that is applied, previouslt it was an INFO class which displays a blue color, but this was wrong, it should have been red indicating an error.